### PR TITLE
Update to Zewo 0.13

### PR DESCRIPTION
**Pull request by joernroeder**
Tries to update package to use Zewo 0.13, swift 3.0

### CODE REVIEW

File: Sources/SQL/Core/ConnectionProtocol.swift [hunk 1]
```diff
@@ -36,13 +36,13 @@ public protocol ConnectionInfoProtocol {
...
     
-    init?(uri: URI)
+    init?(uri: URL)
 }
 
 public protocol ConnectionProtocol: class {
     associatedtype InternalStatus
     associatedtype Result: ResultProtocol
-    associatedtype Error: ErrorProtocol, CustomStringConvertible
+    associatedtype ErrorAndStringConvertible: Error, CustomStringConvertible
```
Comment (robertjpayne) [inline Sources/SQL/Core/ConnectionProtocol.swift]: Any reason you named this ErrorAndStringConvertible? It reads a lot nicer as `Error` even if it is CustomStringConvertible

Comment (Danappelxx) [reply Sources/SQL/Core/ConnectionProtocol.swift]: I assume it's because `Error` is a reserved name by swift now. I'll change it to `ConnectionError`

Comment (Danappelxx) [reply Sources/SQL/Core/ConnectionProtocol.swift]: Fixed :)

**Comment by Danappelxx** [comment]
I rebased on master and fixed a few things. Your part looks good! Once CI passes I'll merge 👍

**Comment by Danappelxx** [comment]
Excellent! Merging

**Pull request merged**